Character Swap

gamingdevotee opened the pull request:
This code allows swapping between characters and cameras using an array. It will only allow swapping between two at the moment but edited a bit it will allow multiple characters.

gamingdevotee committed:
Add files via upload

This code allows swapping between characters and cameras using an array. At the moment it only allows swapping between two characters and cameras but when given a bit more time it can be edited to allow more.

notleeandrew commented:
2 STARS
- works very well.
- it's great that you're able to do this using key commands.

wish
- you didn't give any description as to what I should apply the piece of code to.

gamingdevotee committed:
Add files via upload

This code is a script that allows use of a button to play a door animation (and potentially others). It is set up to have a set distance between the player which prevents the player from opening the door from a set distance away.
